docs/support/04-browser-support.md

```diff
@@ -0,0 +1,15 @@
+## Browser Support
+
+CARTO.js works for the last 2 versions in any modern browser and IE11.
```

Comment:
Copy&Paste, _ejem_ 😂 

Why don't we put the same ones as in https://github.com/CartoDB/airship/blame/master/docs/guides/05-web-components.md#L63? We need to be consistent.